src/main/scala/no/ndla/conceptapi/controller/InternController.scala

```diff
@@ -126,6 +132,36 @@ trait InternController {
       }
     }
 
+    get("/dump/draft-concept/") {
```

Comment:
Trur egentlig ikkje vi treng endepunkt for å dumpe alle concepts, men skader ikkje å ha det med.